Sources/PointFree/Account/Account.swift

```diff
@@ -13,7 +13,17 @@ let accountResponse =
   filterMap(require1 >>> pure, or: loginAndRedirect)
     <| fetchAccountData
     >-> writeStatus(.ok)
-    >-> respond(accountView.contramap(lower), layout: simplePageLayout(title: "Account", currentUser: get5))
+    >-> map(lower)
+    >>> respond(
+      view: accountView,
+      layoutData: { subscription, teamInvites, teammates, emailSettings, currentUser in
```

Comment:
There's a lot of noise here, but maybe that's OK. Wonder about trying something shorter like:

``` swift
layoutData: { .init(currentUser: $0.4, data: $0, title: "Account" }
```